### PR TITLE
us1287

### PR DESCRIPTION
**Pull request by IngageMaddoxT**

### CODE REVIEW

File: Gateway/crds-angular/Controllers/API/VolunteerApplicationController.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        //            return Ok(family);
+        //        }
+        //        catch (Exception exception)
+        //        {
+        //            var apiError = new ApiErrorDto("Volunteer Application GET Family", exception);
+        //            throw new HttpResponseException(apiError.HttpResponseMessage);
+        //        }
+        //    });
+        //}
+    }
```
Comment (andycanterbury) [inline Gateway/crds-angular/Controllers/API/VolunteerApplicationController.cs]: Remove commented method.

File: Gateway/crds-angular/Services/PersonService.cs [hunk 2]
```diff
@@ -47,6 +47,40 @@ public List<Skill> GetLoggedInUserSkills(int contactId, string token)
             return GetSkills(contactId, token);
         }
 
+        public Person GetPerson(int contactId)
```
Comment (andycanterbury) [inline Gateway/crds-angular/Services/PersonService.cs]: Why aren't we just returning the Contact object if it has all the stuff in the right format?

Comment (IngageMaddoxT) [inline Gateway/crds-angular/Services/PersonService.cs]: Cause we don't want to expose MP objects in Gateway

**Comment by andycanterbury** [comment]
:ok_hand:

**Pull request merged**